src/kirby/directives/nativescript-only/elevation/elevation.directive.tns-only.ts

```diff
@@ -0,0 +1,52 @@
+import { Directive, OnInit, Input, ElementRef, Renderer2 } from '@angular/core';
+import { isIOS } from 'tns-core-modules/ui/page/page';
+
+import { ScssHelper } from '../../../scss/scss-helper';
+
+@Directive({ selector: '[kirbyElevation]' })
+export class NativeScriptElevationDirective implements OnInit {
+  @Input() kirbyElevation?: 'z2' | 'z4' | 'z8';
+  @Input() iosMasksToBounds?: boolean | string;
+
+  elevation: number;
+
+  constructor(private el: ElementRef, private renderer: Renderer2) {}
+
+  ngOnInit(): void {
+    if (!this.kirbyElevation) {
+      return;
+    }
+
+    // Due to a timing issue, the native element is not available initially:
+    // https://github.com/NativeScript/nativescript-angular/issues/848
+    // As a workaround we add 50 ms delay.
+    setTimeout(this.applyElevation.bind(this), 50);
```

Comment:
50 ms is a magic number, that might not work in all circumstances.
So better to check every 5 or 10 ms and then call recursively for a number of times if the view hasn't been initialized yet...